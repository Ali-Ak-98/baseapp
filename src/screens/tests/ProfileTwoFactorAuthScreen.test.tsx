// import { shallow } from 'enzyme';
// import * as React from 'react';
// import { InjectedIntlProps } from 'react-intl';
// import { connect, Provider } from 'react-redux';
// import { ProfileTwoFactorAuthScreen } from '../';
// import { store } from '../../redux';

// const ProfileTwoFactorAuthTab = connect()(ProfileTwoFactorAuthScreen);

// const setup = (props: Partial<InjectedIntlProps> = {}) =>
//     shallow(
//         <Provider store={store}>
//             <ProfileTwoFactorAuthTab/>
//         </Provider>,
//     );

// deprecated
describe('ProfileTwoFactorAuthScreen test', () => {    
    // deprecated: imposible to test js files
    it('test', () => {
        expect(true).toBeTruthy();
    });
    // it('should render', () => {
    //     const wrapper = setup();
    //     expect(wrapper).toMatchSnapshot();
    //     expect(wrapper).toBeDefined();
    // });
});
