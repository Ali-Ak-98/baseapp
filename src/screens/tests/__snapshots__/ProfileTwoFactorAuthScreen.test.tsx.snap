// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileTwoFactorAuthScreen test should render 1`] = `<Connect(InjectIntl(withRouter(Connect(ToggleTwoFactorAuthComponent)))) />`;
